src/header.rs

```diff
@@ -5,13 +5,14 @@
 
 use crate::error::Error;
 use byteorder::{BigEndian, ByteOrder};
-use std::io::{Seek, SeekFrom, Read};
+use std::io::{Read, Seek, SeekFrom};
 
 #[derive(Debug, Default)]
 pub struct Header {
     pub name: String,
     pub size: usize,
     pub start: u64,
+    pub end: u64,
```

Comment:
As we have the `start` and the `size`, the `end` can be calculated: `start + end`. This field and its method is not necessary.

Reply:
Yeah, that's understood but most of the well-known parsers on crates.io print these minimum 4 details in the parsing output. Is there any problem with printing these values?

Reply:
You can print the `end` value, but it does not mean you need a field to keep the `end` value while you can calculate it. Whenever you need to print it: `end = start + size`

Reply:
I noticed one of your older comments on this, the newer commit on the stco-parsing branch removes the end() method and uses the above as suggested.